sparse/src/KokkosSparse_spmv.hpp

```diff
@@ -752,7 +752,7 @@ void spmv_struct(const ExecutionSpace& space, const char mode[], const int stenc
                                   typename AMatrix::const_size_type>
       AMatrix_Internal;
 
-  AMatrix_Internal A_i = A;
```

Comment:
What happened here?

Reply:
Oops, I didn't mean to commit that here. That is #2629 and it was needed to let my local build succeed.

Reply:
I just reverted that line.